src/App.tsx

```diff
@@ -96,13 +96,7 @@ export default class App extends React.Component<unknown, AppState> {
           </Container>
         </main>
         <footer>
-          <Navbar bg="dark" variant="dark" expand="lg" fixed="bottom">
-            <Container>
-              <span className="text-muted">
-                &copy; 2022 The Brigade Authors
-              </span>
-            </Container>
-          </Navbar>
```

Comment:
Kindly tell me why you changed these lines to a single line ? What was the issue with this ?

Reply:
Earlier, the footer was using the `Navbar` component from react-bootstrap which is actually semantically incorrect (i.e. using a Navbar component for the footer). Besides, the footer didn't have much content. Thus, I removed all the code and used only a simple `footer` element :)

Reply:
A few things here.

I'm not sure where the idea comes from that using a navbar in the footer is semantically incorrect. Bootstrap's own official examples include numerous footers that contain navbars:

https://getbootstrap.com/docs/5.1/examples/footers/

Beyond that, a navbar was used in the footer to account for the high probability of the footer being expanded to include a variety of different links.

All of that having been said, I don't really want to waste a lot of time arguing over whether the navbar belongs or not...

The _container_ element certainly should not go away because it affects the appearance of the page. As in the case of the top navbar and the main body of the page, the container element was used in the footer to constrain content closer to the center of larger viewports.

If you look at the deployment preview in Netlify, you will see this has now changed.